src/core/agent/services/keriaNotificationService.ts

```diff
@@ -940,18 +947,27 @@ class KeriaNotificationService extends AgentService {
             .get(admitExchange.exn.p);
           const credentialId = grantExchange?.exn?.e?.acdc?.d;
           const credentialMetadata =
-              await this.credentialStorage.getCredentialMetadata(credentialId);
+            await this.credentialStorage.getCredentialMetadata(credentialId);
           const credential = await this.props.signifyClient
             .credentials()
-            .get(credentialId);
+            .get(credentialId)
+            .catch((error) => {
+              const status = error.message.split(" - ")[1];
```

Comment:
Can we revert this particular change? I would prefer if it just throws the error here as it should exist.

It will throw an error on the next line anyway with `credential.status.s` as it'll be undefined. So lets just revert.